src/content/en/2021/webassembly.md

```diff
@@ -476,18 +479,18 @@ Let's take a look at their adoption in the Almanac dataset too:
 
 One feature stands out—it's the <a hreflang="en" href="https://github.com/WebAssembly/sign-extension-ops/blob/master/proposals/sign-extension-ops/Overview.md">sign-extension operators proposal</a>. It was shipped in all browsers not too long after the MVP, and enabled in LLVM (a compiler backend used by Clang / Emscripten and Rust) by default, which explains its high adoption rate. All other features currently have to be enabled explicitly by the developer at compilation time.
 
-For example, <a hreflang="en" href="https://github.com/WebAssembly/nontrapping-float-to-int-conversions/blob/master/proposals/nontrapping-float-to-int-conversion/Overview.md">non-trapping float-to-int conversions</a> is very similar in spirit to sign-extension operators—it also provides built-in conversions for numeric types to save some code size—but it became uniformly supported only recently with the release of Safari 15. That's why this feature is not yet enabled by default, and most developers don't want the complexity of building & shipping different versions of their WebAssembly module to different browsers without a very compelling reason. As a result, none of the Wasm modules in the dataset used those conversions.
+For example, <a hreflang="en" href="https://github.com/WebAssembly/nontrapping-float-to-int-conversions/blob/master/proposals/nontrapping-float-to-int-conversion/Overview.md">non-trapping float-to-int conversions</a> is very similar in spirit to sign-extension operators - it also provides built-in conversions for numeric types to save some code size - but it became uniformly supported only recently with the release of Safari 15. That's why this feature is not yet enabled by default, and most developers don't want the complexity of building and shipping different versions of their WebAssembly module to different browsers without a very compelling reason. As a result, none of the Wasm modules in the dataset used those conversions.
 
-Other features with zero detected usages—multi-value, reference types and tail calls—are in a similar situation: they could also benefit most WebAssembly use-cases, but they suffer from incomplete compiler and/or engine support.
+Other features with zero detected usages - multi-value, reference types and tail calls—are in a similar situation: they could also benefit most WebAssembly use-cases, but they suffer from incomplete compiler and/or engine support.
```

Comment:
Same question about changes to dash - I think @tunetheweb changed them from ` - ` to mdashes (`—`) in his round of review, now those edits are changing them back and I'm feeling lost about which one is right 😀

Reply:
Oh shoot lol, hope I didn’t make it worse

Reply:
[Style guide](https://developers.google.com/style/dashes) says spaceless em-dashes:

> **Em dashes**
> To indicate a break in the flow of a sentence—or an interruption—use an em dash, also known as a long dash. Don't put a space before or after it.

I hate that and think they should have spaces, but that's the style guide we're following!

Reply:
> I hate that 
YES

Reply:
How do I uncommit all of them?

Reply:
Let me do a quite search and replace to change them back. Two seconds.

> > I hate that
> YES

I think spaceless em-dashes are more common in America. And unfortunately it's an Google (so I guess American?) style guide.

Reply:
Put the evil spaceless em-dashes back in: https://github.com/HTTPArchive/almanac.httparchive.org/pull/2605/commits/c60166f8b730f3788c3091803d32b051a10494d6